### ISSUE TITLE
Function listing should support 

### ISSUE THREAD

**Issue by mcclure** [bjornbytes/lovr-docs#118]
Since 0.16 or so the lovr-docs repo has listed functions with more than one legal overload with this javascript picker at the top:

https://lovr.org/docs/v0.16.0/lovr.graphics.newBuffer

I love this, before I kept tripping myself up not noticing overloads lower in the file and the new design is clean and convenient.

There is one problem. Because all the overloads are in one page, it doesn't seem to be possible to link to a single one anymore. [this is sometimes inconvenient](https://github.com/bjornbytes/lovr-docs/issues/117).

IMO, clicking one of the overloads should use JavaScript to set the #anchor (so that links to the current overload can be copied) and set javascript `document.history`. I have worked with this JS API before and would be happy to implement this change myself if you could point me to how/where the overload directory is set (in a script in this repo? in an external tool?)

**Comment by bjornbytes** [comment]
Yes, I really want to add this but wasn't immediately able to figure it out while implementing it.

It uses a [CSS-only tab system](https://github.com/bjornbytes/lovr.org/blob/master/css/docs.css#L383-L436) implemented via [radio buttons](https://github.com/bjornbytes/lovr.org/blob/master/.lua/generate.lua#L364-L392).  The JS is in `js/docs.js` in that repo.
